internal/core/src/exec/expression/UnaryExpr.h

```diff
@@ -156,6 +178,32 @@ struct UnaryIndexFunc {
                          proto::plan::OpType::PrefixMatch);
             dataset->Set(milvus::index::PREFIX_VALUE, val);
             return index->Query(std::move(dataset));
+        } else if constexpr (op == proto::plan::OpType::Match) {
+            if constexpr (!std::is_same_v<T, std::string_view> &&
```

Comment:
wrapper this add module to a  new function is better, same as ElementFunc

Reply:
OK.